assets/elm/Search.elm

```diff
@@ -205,7 +222,16 @@ renderDropdownItems model dropdownItem =
     else
         option [ Html.Attributes.value dropdownItem ] [ text dropdownItem ]
 
-
+renderSearch : (String -> Msg) -> Html Msg
```

Comment:
Would be good to make this look the same as the search-venue page:
<img width="541" alt="screen shot 2018-11-30 at 09 47 49" src="https://user-images.githubusercontent.com/8939909/49281606-e2a31880-f484-11e8-8242-d3b2c34924f5.png">
<img width="465" alt="screen shot 2018-11-30 at 09 48 18" src="https://user-images.githubusercontent.com/8939909/49281613-e5057280-f484-11e8-8f27-d2574ba60c57.png">